js/data/locale/my/dateformats.json

```diff
@@ -68,88 +68,88 @@
-                "hmz": "z B h:mm",
-                "ahmz": "z hB h:mm",
-                "ahms": "hB h:mm:ss",
-                "hmsz": "z B h:mm:ss",
-                "ahmsz": "z hB h:mm:ss"
+                "ahm": "a h:mm",
+                "hms": "a h:mm:ss",
+                "hmz": "z a h:mm",
+                "ahmz": "z a h:mm",
+                "ahms": "a h:mm:ss",
+                "hmsz": "z a h:mm:ss",
+                "ahmsz": "z a h:mm:ss"
             },
             "24": {
                 "h": "HH",
                 "m": "mm",
                 "s": "ss",
                 "ah": "HH",
-                "hm": "B H:mm",
+                "hm": "H:mm",
                 "ms": "mm:ss",
-                "ahm": "B H:mm",
-                "hms": "B HH:mm:ss",
-                "hmz": "z B H:mm",
-                "ahmz": "z B H:mm",
-                "ahms": "B HH:mm:ss",
-                "hmsz": "z B HH:mm:ss",
-                "ahmsz": "z B HH:mm:ss"
+                "ahm": "H:mm",
+                "hms": "HH:mm:ss",
+                "hmz": "z H:mm",
+                "ahmz": "z H:mm",
+                "ahms": "HH:mm:ss",
+                "hmsz": "z HH:mm:ss",
+                "ahmsz": "z HH:mm:ss"
             }
         },
         "range": {
             "c00": {
                 "f": "{sy}၊ {sm} {sd} {st} – {et}",
                 "l": "{sy}၊ {sd} {sm} {st} – {et}",
                 "m": "{sy}၊ {sm} {sd} {st} – {et}",
-                "s": "{sd}-{sm}-{sy} {st} – {et}"
+                "s": "{sd}-{sm}-{sy}၊ {st} – {et}"
             },
             "c01": {
                 "f": "{sy}၊ {sm} {sd} {st} – {ey}၊ {em} {ed} {et}",
                 "l": "{sy}၊ {sd} {sm} {st} – {ey}၊ {ed} {em} {et}",
                 "m": "{sy}၊ {sm} {sd} {st} – {ey}၊ {em} {ed} {et}",
-                "s": "{sd}-{sm}-{sy} {st} – {ed}-{em}-{ey} {et}"
+                "s": "{sd}-{sm}-{sy}၊ {st} – {ed}-{em}-{ey}၊ {et}"
             },
             "c02": {
                 "f": "{sy}၊ {sm} {sd} {st} – {ey}၊ {em} {ed} {et}",
                 "l": "{sy}၊ {sd} {sm} {st} – {ey}၊ {ed} {em} {et}",
                 "m": "{sy}၊ {sm} {sd} {st} – {ey}၊ {em} {ed} {et}",
-                "s": "{sd}-{sm}-{sy} {st} – {ed}-{em}-{ey} {et}"
+                "s": "{sd}-{sm}-{sy}၊ {st} – {ed}-{em}-{ey}၊ {et}"
             },
             "c03": {
                 "f": "{sy}၊ {sm} {sd} {st} – {ey}၊ {em} {ed} {et}",
                 "l": "{sy}၊ {sd} {sm} {st} – {ey}၊ {ed} {em} {et}",
                 "m": "{sy}၊ {sm} {sd} {st} – {ey}၊ {em} {ed} {et}",
-                "s": "{sd}-{sm}-{sy} {st} – {ed}-{em}-{ey} {et}"
+                "s": "{sd}-{sm}-{sy}၊ {st} – {ed}-{em}-{ey}၊ {et}"
             },
             "c10": {
                 "f": "{sy}၊ {sm} {sd} – {ed}",
                 "l": "{sy}၊ {sd} {sm} – {ed} {em}",
                 "m": "{sy}၊ {sm} {sd} – {ed}",
-                "s": "{sd} – {ed}-{em}-{ey}"
+                "s": "{sd} – {ed}-{em}-{ey}၊"
```

Comment:
I looked up what that symbol meant, and it looks like it is the Myanmar equivalent of a comma. So, we have to take it back out again for any format where it is used at the end like this. Sorry for the confusion!

Reply:
I updated the data again and fixed timezone info too. Please take a look again at the latest change.